bots/naughty-prune

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# This file is part of Cockpit.
+#
+# Copyright (C) 2017 Red Hat, Inc.
+#
+# Cockpit is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation; either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# Cockpit is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+
+# To use this example add a line to an issue with the "bot" label
+#
+#  * [ ] naughty-prune
+#
+
+import math
+import os
+import sys
+import subprocess
+import time
+
+sys.dont_write_bytecode = True
+
+import task
+
+# Number of days after which a known issue is treated as stale
+DAYS = 14
+
+BOTS = os.path.abspath(os.path.dirname(__file__))
+BASE = os.path.normpath(os.path.join(BOTS, ".."))
+
+def execute(*args):
+    if task.verbose:
+        sys.stderr.write("+ " + " ".join(args) + "\n")
+    return subprocess.check_output(args, cwd=BASE)
+
+def run(unused, verbose=False, **kwargs):
+
+    # Since a month ago
+    since = time.time() - (DAYS * 86400)
```

Comment:
The comment seems to be from an older version.

I prefer spelling the 3600 * 24 out to make clear that it's seconds per day. But maybe I should just know the number of seconds in a day ;)